pkg/models/v1/generate.go

```diff
@@ -0,0 +1,3 @@
+package models
+
+//go:generate protoc -I=$GOPATH/src/github.com/trisacrypto/trisa/proto -I=$GOPATH/src/github.com/trisacrypto/directory/proto --go_out=. --go_opt=module=github.com/trisacrypto/directory/pkg/models/v1 --go-grpc_out=. --go-grpc_opt=module=github.com/trisacrypto/directory/pkg/models/v1 gds/models/v1/models.proto
```

Comment:
Should this be 

```
//go:generate protoc -I=$GOPATH/src/github.com/trisacrypto/trisa/proto -I=$GOPATH/src/github.com/trisacrypto/directory/proto --go_out=. --go_opt=module=github.com/trisacrypto/directory/pkg/models/v1 --go-grpc_out=. --go-grpc_opt=module=github.com/trisacrypto/directory/pkg/models/v1 pkg/models/v1/models.proto
```
?

Reply:
The last argument refers to the location of the proto file, however I suppose we could also move the file up to the top level in `proto/` to be consistent with the package layout.